guides/user/managed-delivery/environment-constraints/index.md

```diff
@@ -0,0 +1,319 @@
+
+`beginAnalysisAfter`: *(Optional)*; Type: `Duration`; Default Value: `PT10M`
+
+After the deployment of canary clusters completes, Spinnaker will wait this amount of time before launching the 
+first Kayenta analysis. This allows for the filtering or inclusion of service warmup behavior in the analysis. 
+
+`canaryAnalysisInterval`: *(Optional)*; Type: `Duration`; Default Value: `PT30M`
+
+After the completion of any `beginAnalysisAfter` delay, Kayenta analysis are launched at this recurring interval, 
+until post-`beginAnalysisAfter` runtime exceeds the `lifetime` parameter.
+
+`lifetime`: *(Optional)*; Type: `Duration`; Default Value: `PT30M`
+
+Maximum runtime for the canary, exempting deployment times and any initial delay configured via `beginAnalysisAfter`.
+
+`cleanupDelay`: *(Optional)*; Type: `Duration`; Default Value: `0`
+
+When a canary fails due to a failing Kayenta score, Spinnaker will wait this amount of time before deleting canary 
+clusters. Useful if engineers desire an opportunity to manually inspect instances involved in the failure.
+
+`marginalScore`: Type: `Integer`
+
+If Kayenta generates a canary score below the `marginalScore`, it is considered an immediate failure.
+
+`passScore`: Type: `Integer`
+
+If Kayenta generates a final canary score >= `passScore`, the constraint will pass.
+
+`regions`: Type: `Set<String>` 
+
+Each region will get its own independently deployed and analyzed control/experiment pair.
+
+`capacity`: Type: `Integer`
+
+Control and experiment clusters have their min/max/desired pinned to this value.
+
+`source`: Type: `CanarySource`
+
+Defined in yaml as a `Map<String, String>` with the following properties:
+  * `account`
+  * `cloudProvider`
+  * `cluster`
+
+This should refer to an active cluster in the target environment deployed in
+all of the regions the constraint is configured to run canaries in. Control/experiment clusters are configured
+based on the source cluster (i.e. account, subnet, instance type, firewall rules).
+
+`minSuccessfulRegions`: *(Optional)*; Type: `Integer`; Default Value: `0`
+
+If set to >0 on a multi-region canary, the constraint will pass if Kayenta analysis passes in this many regions. 
+I.e. if set to 2 on a constraint that defines a 3-region canary, the failure of a single region is ignored.
+
+`failureCancelsRunningRegions`: *(Optional)*; Type: `Boolean`; Default Value: `true`
+
+If true, the failure of any region
+that violates the `minSuccessfulRegions` parameter triggers the immediate cancellation of any canaries still
+running in other regions. 
+
+`metricsAccount`: *(Optional)*; Type: `String` 
+
+If set, overrides the `defaults.constraint.canary.metrics-account` property. 
+This may be required in environments with multiple TSD instances.
+
+`storageAccount`: *(Optional)*; Type: `String`
+
+If set, overrides the `defaults.constraint.canary.storage-account` property. Not typically required.
```

Comment:
Could you provide any guidance for when this _would_ be required? Or else, maybe, just delete the "Not typically required." on the assumption that the reader understands that we're just overriding the default storage account, and she'd know when to do so.